src/ansys/geometry/core/plotting/plotter_helper.py

```diff
@@ -263,6 +263,17 @@ def add(self, object: Any, **plotting_options):
         """
         self._pl.add(object=object, **plotting_options)
 
+    def add_list(self, object_list: List[Any], **plotting_options):
```

Comment:
Are you using this anywhere?

Reply:
This was meant to be used by users directly, but I reworked this to be more intuitive, by accepting lists in the `add` method. Usage example:

```python
ph = PlotterHelper(use_trame=True)
ph.add(plot_list)
view = TrameVisualizer()
view.set_scene(plotter=ph)
view.show()

```